src/EppoClient.php

```diff
@@ -70,48 +81,50 @@ public static function init(
         RequestFactoryInterface $requestFactory = null,
         ?bool $isGracefulMode = true
     ): EppoClient {
-        if (self::$instance === null) {
```

Comment:
no more singleton pattern.

Reply:
This diff looks way worse than it is. un-indented the block and it's having trouble tracking.